src/Uno.UI/Helpers/TypeMappingHelper.cs

```diff
@@ -72,6 +91,32 @@ internal static void ClearMappings()
 		{
 			MappedTypeToOrignalTypeMapings.Clear();
 			OriginalTypeToMappedType.Clear();
+			AllMappedTypeToOrignalTypeMapings.Clear();
+			AllOriginalTypeToMappedType.Clear();
+		}
+
+		private static TaskCompletionSource _pauseReloadingCompletion;
+
+		internal static Task WaitToReload()
```

Comment:
`WaitForReload` ? Not sure :)

Reply:
Yeh I think the term for updating the UI will be something like "UI Update". So this would become "WaitForUIUpdate"